compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -585,6 +585,22 @@ abstract class PrepJSInterop extends plugins.PluginComponent
       if (shouldCheckLiterals)
         checkJSNameLiteral(sym)
 
+      // Check for overrides with different JS names - issue #1983
+      for (overridingPair <- new overridingPairs.Cursor(sym).iterator) {
```

Comment:
In 2.10 the `Cursor` doesn't have the `iterator` method, but at least it exists.